### PR TITLE
Added vaccinations

### PR DESCRIPTION
**Pull request by reagan-meant**
@rbuisson Could you help me proof read this as  WIP

### CODE REVIEW

File: configuration/concepts/vaccination_history.csv [hunk 1]
```diff
@@ -2,25 +2,32 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully sp
...
-5ba4e40a-28f9-4c21-b589-add0c9cdec78,,CIEL:83531,Rotavirus Vaccine,Rotavirus vaccine,Vaccin rotavirus,Rota,Procedure,N/A,,,,,,,,,
+5ba4e40a-28f9-4c21-b589-add0c9cdec78,,CIEL:83531,Rotavirus Vaccine,RV,Vaccin rotavirus,Rota,Procedure,N/A,,,,,,,,,
 19e9c51d-e0de-4b2c-8eb0-598dbd11187c,,CIEL:1685,Pentavalent Vaccine,Pentavalent Vaccine,Pentavalent vaccine,Penta,Procedure,N/A,,,,,,,,,
-67b20a95-eaa4-4a54-ba8c-2e3c8aa2f327,,CIEL:82215,Pneumococcal Vaccine,Pneumococcal Vaccine,Vaccin pneumocoque,Pneumo,Procedure,N/A,,,,,,,,,
-9f24e11b-24e7-4a44-917b-645a461defc3,,CIEL:781,Diphtheria Tetanus Pertusis Vaccine,DTP,Vaccin diphtérique tétanique coquelucheux,DT Coqueluche (DTP),Procedure,N/A,,,,,,,,,
+67b20a95-eaa4-4a54-ba8c-2e3c8aa2f327,,CIEL:82215,Pneumococcal Vaccine,PCV13,Vaccin pneumocoque,PCV13,Procedure,N/A,,,,,,,,,
+67b20a95-eaa4-4a54-ba8c-2e3c8aa2f327,,,Pneumococcal polysaccharide,PPSV23,Polysaccharide antipneumococcique,PPSV23,Procedure,N/A,,,,,,,,,
+9f24e11b-24e7-4a44-917b-645a461defc3,,CIEL:781,Diphtheria Tetanus Pertusis Vaccine,DTP,Vaccin diphtérique tétanique coquelucheux,DTP,Procedure,N/A,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/vaccination_history.csv]: Let's keep as "DT Coqueluche (DTP)"

File: configuration/concepts/vaccination_history.csv [hunk 2]
```diff
@@ -2,25 +2,32 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully sp
...
 3afaa3ce-0dbe-4ac8-964a-1913ee930d34,,CIEL:17,Diphtheria Tetanus Vaccine,DT,Vaccin diphtérique tétanique,dT,Procedure,N/A,,,,,,,,,
 0a28eec8-0353-415f-9eee-6f47ff67f4ce,,CIEL:162586,Measles Rubella Vaccine,MR,Vaccin rubéole rougeole,RR,Procedure,N/A,,,,,,,,,
 d5b69086-c2b0-4ef7-8a75-a5f5ead221fc,,CIEL:159701,Measles Mumps Rubella Vaccine,MMR,Vaccin rougeole rubéole orreillons,ROR,Procedure,N/A,,,,,,,,,
 7b34a0c2-a88b-4409-b654-d723e70eccd0,,CIEL:159704,Chickenpox vaccine,VZV,Vaccin varicelle,VZV,Procedure,N/A,,,,,,,,,
 70bc9209-2b6e-4e17-9c4d-58923d3ac49e,,CIEL:77423,Hepatitis A Vaccine,HepA Vaccine,Vaccin hépatite A,Vaccin hépatite A,Procedure,N/A,,,,,,,,,
 0146bbd0-0128-4c7d-9bbb-d353c80c03f2,,CIEL:159708,Human papillomavirus vaccine,HPV Vaccine,Vaccin papillomavirus humain,HPV,Procedure,N/A,,,,,,,,,
-ebcc1391-8d2c-4e4d-b879-c24e89e94130,,CIEL:159900,Meningococcal Vaccine,Meningococcal Vaccine,Vaccin infections invasives à méningocoque,Vaccin meningococcal,Procedure,N/A,,,,,,,,,
+ebcc1391-8d2c-4e4d-b879-c24e89e94130,,CIEL:159900,Meningococcal Vaccine,MenACWY-D,Vaccin infections invasives à méningocoque,MenACWY-D,Procedure,N/A,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/vaccination_history.csv]: Let's name it "MenACWY"

File: configuration/concepts/vaccination_history.csv [hunk 3]
```diff
@@ -2,25 +2,32 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully sp
...
+63916e88-4d0e-4bce-8cee-b9f9d29b0a4c,,,Meningococcal B Vaccine,MenB,Vaccin contre le méningocoque B,MenB,Procedure,N/A,,,,,,,,,
 b4c138d6-1ab9-4a0c-9b9d-f46181fafeca,,CIEL:77428,Hepatitis B Vaccine,HepB Vaccine,Vaccin hépatite B,Vaccin hépatite B,Procedure,N/A,,,,,,,,,
-407b036e-c49e-46a4-befe-b0bb9de83dc0,,CIEL:78032,Influenza Virus Vaccine,Influenza Virus Vaccine,Vaccin grippe,Vaccin grippe,Procedure,N/A,,,,,,,,,
+87acf428-7c02-42e9-ab61-84677e543afd,,,Influenza Virus(Inactivated) Vaccine,IIV,Vaccin contre le virus de la grippe (inactivé),IIV,Procedure,N/A,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/vaccination_history.csv]: Replace "Vaccin contre le virus de la grippe" by "Vaccin antigrippal"

File: configuration/concepts/vaccination_history.csv [hunk 4]
```diff
@@ -2,25 +2,32 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully sp
...
 83189d40-6746-4bb6-a987-9afd4a766690,,CIEL:165907,Vaccine lot expiration date,Vaccine lot expiration date,Date de péremption du vaccin,Date de péremption du vaccin,Finding,Date,,,,,,,,,
 beab4061-e63d-498a-ac08-a19ced63d074,,CIEL:166011,Immunization non-coded,Immunization non-coded,Immunization non-coded,Immunization non-coded,Question,Text,,,,,,,,,
-38c6fbba-9185-4b80-9bd7-fa46052c22db,,CIEL:163100,Procedure received by patient,Procedure received by patient ,Procédure reçue par le patient,Procédure reçue par le patient,Question,Coded,,Yes; No,,,,,,,
-ee938d18-fa98-43e3-809f-61aa29726a46,,CIEL:1421,Vaccination History,Vaccination History,Historique vaccinale,Historique vaccinale,ConvSet,N/A,,,Vaccinations; Vaccination date; Vaccination sequence number; Vaccine Manufacturer; Vaccine Lot Number; Vaccine lot expiration date;,,,,,,
+38c6fbba-9185-4b80-9bd7-fa46052c22db,,CIEL:163100,Procedure received by patient,"Procedure received by patient ",Procédure reçue par le patient,Procédure reçue par le patient,Question,Coded,,Yes; No,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/vaccination_history.csv]: Let's remove the space " ".

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/concepts/vaccination_history.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/vaccination_history.csv]: Done

File: configuration/concepts/vaccination_history.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/vaccination_history.csv]: Done

File: configuration/concepts/vaccination_history.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/vaccination_history.csv]: Done

**Pull request merged**